easybuild/easyblocks/generic/juliapackage.py

```diff
@@ -96,11 +96,12 @@ def install_step(self):
         """Install Julia package with Pkg"""
 
         # prepend installation directory to Julia DEPOT_PATH
+        # extensions in a bundle can share their DEPOT_PATH
         # see https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_DEPOT_PATH
-        depot_path = os.getenv('JULIA_DEPOT_PATH')
-        if depot_path is not None:
-            depot_path = ':'.join([self.installdir, depot_path])
-        env.setvar('JULIA_DEPOT_PATH', depot_path)
+        depot_path = os.getenv('JULIA_DEPOT_PATH', '')
+        if self.installdir not in depot_path:
+            depot_path = ':'.join([depot for depot in (self.installdir, depot_path) if depot])
```

Comment:
please use `os.pathsep` instead of `:`

Reply:
fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/2869/commits/f2b4131c0821019b47d96568fc604ee0bcf20de0